gamemodes/sss/core/vehicle/trunk.pwn

```diff
@@ -139,30 +139,16 @@ VehicleTrunkUpdateSave(playerid)
 {
 	if(IsValidVehicle(trunk_CurrentVehicle[playerid]))
 	{
-		new owner[MAX_PLAYER_NAME];
-
-		GetVehicleOwner(trunk_CurrentVehicle[playerid], owner);
-
-		if(!isnull(owner))
-		{
-			new name[MAX_PLAYER_NAME];
-
-			GetPlayerName(playerid, name, MAX_PLAYER_NAME);
-
-			if(!strcmp(owner, name))
```

Comment:
Why was this check removed? This is just a safety check to ensure the vehicle associated with the player is actually associated back to that player - there were some corner cases I think where vehicles could end up with the wrong player assigned to them. Though these should really be fixed instead.

Reply:
Because if you don't own the vehicle and remove an item, it will come back when it respawns.